src/base/QXmppMessage.cpp

```diff
@@ -1255,14 +1255,16 @@ std::optional<QXmppMessageReaction> QXmppMessage::reaction() const
 ///
 /// Sets a reaction to a message as defined by \xep{0444, Message Reactions}.
 ///
-/// The corresponding storage hint is automatically set.
+/// The corresponding hint must be set manually:
+/// \code
+/// QXmppMessage::addHint(QXmppMessage::Store)
```

Comment:
The function isn't static, so better use it on an object.

Reply:
It was meant as a reference to the method but I adjusted it to make it clear.